core/src/main/scala/zio/config/ReadError.scala

```diff
@@ -132,6 +169,33 @@ sealed trait ReadError[A] extends Exception { self =>
       }
     }).mkString(System.lineSeparator())
   }
+
+  def size: Int =
+    self match {
+      case ReadError.MissingValue(_, _, _)    => 1
+      case ReadError.FormatError(_, _, _, _)  => 1
+      case ReadError.ConversionError(_, _, _) => 1
+      case ReadError.OrErrors(list, _)        => list.map(_.size).sum
+      case ReadError.ZipErrors(list, _)       => list.map(_.size).sum
+      case ReadError.ListErrors(list, _)      => list.map(_.size).sum
+      case ReadError.MapErrors(list, _)       => list.map(_.size).sum
+      case ReadError.Irrecoverable(list, _)   => list.map(_.size).sum
+    }
+
+  def sizeOfZipAndOrErrors: Int =
```

Comment:
This one too seems special-purpose.

Reply:
Sure, I forgot to move this to Read Module.

Reply:
This will also go into the `fold` function